src/lbp.jl

```diff
@@ -0,0 +1,92 @@
+function lbp_original(bit_pattern::Array{Bool, 1})
+	sum([b * 2 ^ (i - 1) for (i, b) in enumerate(bit_pattern)])
+end
+
+UNIFORM_LBP_TABLE = Dict{Array{Bool, 1}, Int}()
+UNIFORM_PATTERN_COUNT = 0
```

Comment:
If this is going to be incremented, you can't make it `const`. For performance you might want to annotate each call site with `::Int`. Alternatively, consider making this a `Ref(0)`.